XenModel/Utils/Helpers.Versions.cs

```diff
@@ -29,89 +29,144 @@
  * SUCH DAMAGE.
  */
 
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
 using XenAdmin.Network;
 using XenAPI;
+using XenCenterLib;
+
 
 namespace XenAdmin.Core

```

Comment:
nit: since you've created a new file, can we also update the namespace? to `XenModel.Utils`

Reply:
Then I should also change the namespace in Helpers.cs because the class is partial, which means many more changes because the class is used all over the place. I don't disagree with the change, but I would leave it for a different PR.